statsmodels/regression/tests/test_mixed_glm.py

```diff
@@ -0,0 +1,350 @@
+    names(df) = c('y', 'g', 'rs', 'x1', 'x2')
+    library(lme4)
+    m = glmer(y ~ 0 + x1 + x2 + (1 + rs| g), family=binomial(), data=df)
+
+    logLik -31.7866107
+    """
+
+    np.random.seed(313)
+    endog, exog, groups, rs = gen_mixed2('Binomial', 20)
+    exog_re = np.vstack((np.ones(len(groups)),rs)).T
+
+    model = MixedGLM(endog, exog, groups=groups, exog_re=np.ones(len(groups)), family=sm.families.Binomial())
+    cmat = np.asarray([[0.54829**2]])
+    fps = np.asarray([0.4996,0.4391])
+    par = MixedGLMParams.from_components(fe_params=fps, cov_re= cmat)
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -38.22342, rtol=1E-5)
+
+    model = MixedGLM(endog, exog, groups=groups, exog_re= exog_re, family=sm.families.Binomial())
+    cmat = np.asarray([[0.65090**2,0.65090*0.74996*0.311],[0.65090*0.74996*0.311,0.74996**2]])
+    fps = np.asarray([0.5950,0.4952])
+    par = MixedGLMParams.from_components(fe_params=fps, cov_re= cmat)
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -37.81756)
+
+def test_laplace_loglike_poisson():
+    """
+    Python:
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+    mat = np.hstack((endog[:, None], groups[:, None], exog))
+    np.savetxt('mat.txt', mat)
+
+    R:
+    df = read.table('mat.txt')
+    names(df) = c('y', 'g', 'x1', 'x2')
+    library(lme4)
+    m = glmer(y ~ 0 + x1 + x2 + (1 | g), family=poisson(), data=df)
+    logLik(m)
+    summary(m)$coef
+    VarCorr(m)
+
+    logLik -85.99506
+    """
+
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+
+    model = MixedGLM(endog, exog, groups=groups, family=sm.families.Poisson())
+    par = MixedGLMParams.from_components(fe_params=np.asarray([1.01505, 0.9464392]),
+                                         cov_re=np.asarray([[0.34564**2]]))
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -85.99506, rtol=1e-5)
+
+def test_laplace_loglike_poisson_sqrt():
+    """
+    Python:
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+    mat = np.hstack((endog[:, None], groups[:, None], exog))
+    np.savetxt('mat.txt', mat)
+
+    R:
+    df = read.table('mat.txt')
+    names(df) = c('y', 'g', 'x1', 'x2')
+    library(lme4)
+    m = glmer(y ~ 0 + x1 + x2 + (1 | g), family=poisson(link=sqrt), data=df)
+    logLik(m)
+    summary(m)$coef
+    VarCorr(m)
+
+    logLik -118.5216
+    """
+
+    np.random.seed(313)
+    endog, exog, groups = gen_mixed('Poisson', 30)
+
+    sqrt_link = sm.families.links.sqrt
+    model = MixedGLM(endog, exog, groups=groups, family=sm.families.Poisson(link=sqrt_link))
+    par = MixedGLMParams.from_components(fe_params=np.asarray([0.5840612,0.5596115]),
+                                         cov_re=np.asarray([[1.2337**2]]))
+    logLik = model.loglike(par)
+    np.testing.assert_allclose(logLik, -118.5216, rtol=1e-4)
+
+def test_laplace_loglike_gamma_log():
+    """
```

Comment:
The docstring mess up the failure report in nose. The failing test shows mainly `Python` for the test function (header), see TravisCI failing test.

In general we use code comments `#`. but here I don't know what the best way would be to avoid having every line start with `#`. (If there is a python expression before, then `"""` would start a string instead of a docstring. Assign string to code?)